frameworks/Swift/perfect/perfect.dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM swift:4.1
+
+ADD . /perfect
+WORKDIR /perfect
+RUN apt update -yqq && apt install -yqq libpq-dev && apt install -y xsltproc docbook-xsl uuid-dev
+RUN apt-get -y install libmysqlclient-dev
```

Comment:
You can probably turn this into:

`RUN apt update -yqq && apt install -yqq libpq-dev xsltproc docbook-xsl uuid-dev libmysqlclient-dev`

I'd also make it a habit to add your apt packages before adding your framework files (`ADD . /perfect`). This way as you're troubleshooting, and changing the files in `./perfect`, your apt packages remain cached by docker.